### DISCUSSION TITLE
Show beatmap name in replay viewer

### DISCUSSION THREAD

**Discussion by nuewink** [ppy/osu#27492] [Ideas] [answered]
Hi. This is a feature that, to my knowledge, doesn't exist on Lazer, but exists on stable. When viewing a replay, it shows a little text about who is playing and what the song and difficulty are called. I couldn't find how to do it one Lazer, and I didn't find anyone making a thread on the discussions tab
![image](https://github.com/ppy/osu/assets/57014208/fc6b779e-9003-4ee1-bf54-31235717b274)

**Comment by Stoppedpuma** [comment, marked as answer]
Using the skin editor (Ctrl + Shift + S) you can use the "BeatmapAttributeText" component to recreate it.

![1709577469](https://github.com/ppy/osu/assets/58333920/9572dc37-6e25-4806-9e3e-93c219a3bf2c)
